layouts/shortcodes/versioned_href.html

```diff
@@ -0,0 +1,2 @@
+{{ $pathFromRoot := .Get "fromRoot" }}
+"{{ .Site.Data.Solo.DocsVersion }}{{ $pathFromRoot }}"
```

Comment:
why the quotes? (especially when layouts/shortcodes/versioned_md_link.md doesn't have them)

Reply:
md urls are stricter than hrefs, I realized along the way.
I'll see if it makes more sense to use a single convention and add the quotes at the call site
worst case, I'll rename "...quoted" and "...unquoted"